layers/+misc/all-the-icons/README.org

```diff
@@ -0,0 +1,73 @@
+#+TITLE: All The Icons layer
+
+#+TAGS: misc|layer
+
+* Table of Contents                                       :TOC_5_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#usage][Usage]]
+
+* Description
+This layer installs and configures All The Icons.
+
+** Features:
+- Enables and configures icons for dired, ivy, and ibuffer when the appropriate layers are enabled.
+- Defines icon transformers for ivy-rich.
+- Enables icons in company-box.
+
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =all-the-icons= to the existing =dotspacemacs-configuration-layers= list in
+this file.
+
```

Comment:
Please provide a comment that users should install all-the-icons-font. I think there is a function available for this in emacs.